packages/flutter_tools/test/general.shard/commands/build_test.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:args/args.dart';
+import 'package:flutter_tools/src/commands/build.dart';
+import 'package:flutter_tools/src/commands/build_aar.dart';
+import 'package:flutter_tools/src/commands/build_apk.dart';
+import 'package:flutter_tools/src/commands/build_appbundle.dart';
+import 'package:flutter_tools/src/commands/build_fuchsia.dart';
+import 'package:flutter_tools/src/commands/build_ios.dart';
+import 'package:flutter_tools/src/commands/build_linux.dart';
+import 'package:flutter_tools/src/commands/build_macos.dart';
+import 'package:flutter_tools/src/commands/build_web.dart';
+import 'package:flutter_tools/src/commands/build_windows.dart';
+
+import '../../src/common.dart';
+import '../../src/context.dart';
+
+void main() {
+  testUsingContext('All build commands support null safety options', () {
+    final List<BuildSubCommand> commands = <BuildSubCommand>[
+      BuildWindowsCommand(verboseHelp: false),
+      BuildLinuxCommand(verboseHelp: false),
+      BuildMacosCommand(verboseHelp: false),
+      BuildWebCommand(verboseHelp: false),
+      BuildApkCommand(verboseHelp: false),
+      BuildIOSCommand(verboseHelp: false),
+      BuildAppBundleCommand(verboseHelp: false),
+      BuildFuchsiaCommand(verboseHelp: false),
+      BuildAarCommand(verboseHelp: false),
```

Comment:
Not sure if BuildIosFrameworks makes sense here

Reply:
Probably spiritually the same as `BuildAarCommand`, right?

Reply:
Ahh yes, but it would need to be updated to support using BuildInfo the same way that BuildAar does.

Reply:
Oh right.  I got that work confused with the assemble work.

Reply:
See https://github.com/flutter/flutter/pull/59874